src/api/webhooks.js

```diff
@@ -44,9 +44,9 @@ export class WebhooksApi extends BaseApi {
    * @param  {AuthCredentials} auth
    * @return {APIResponse}
    */
-  list(auth) {
+  list() {
```

Comment:
@alavers I changed all apis to do that... I hate that I need to do it in all methods... in a ES7 world I'd use decorators instead... but we're not there yet and they are still just a proposal.